examples/using-react-native-web/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "using-react-native-web",
+  "private": true,
+  "description": "Example usage of gatsby-plugin-react-native-web",
+  "author": "Sebastien Lorber <lorber.sebastien@gmail.com>",
+  "dependencies": {
+    "gatsby": "^1.9.221",
+    "gatsby-link": "^1.6.37",
+    "gatsby-plugin-google-analytics": "latest",
+    "gatsby-plugin-react-native-web": "^0.1.2",
+    "gatsby-plugin-react-next": "^1.0.11",
+    "react": "^16.2.0",
+    "react-dom": "^16.2.0",
+    "react-native-web": "^0.5.1"
```

Comment:
We should update this example to use 0.6.x and the latest version of the `gatsby-plugin-react-native-web` now that its using the latest APIs

Reply:
Sure @necolas I just need the changes required to Gatsby to be commited first. I'm in a cyclic dependency problem as I need Gatsby to publish change, so that plugin can be republished, so that example in Gatsby can be updated.

The plugin version I published temporarily was deprecated as I was asked to rename the new Gatsby method and it's not merged yet. I'll make sure the sample use the new plugin version as soon as I can

Reply:
waiting for this PR to be merged: https://github.com/gatsbyjs/gatsby/pull/5017

Reply:
@slorber with the core/plugin updated, lemme know when this is ready to go in 🎉